### ISSUE TITLE
Failed to access docker 

### ISSUE THREAD

**Issue by therexone** [langchain-ai/langgraph-studio#165]
When I start LangGraph Studio, I am getting the following error notification:
```
Command failed with ENOENT: docker info -f json spawn docker ENOENT
```
![image](https://github.com/user-attachments/assets/135b0b7a-5bcf-49b7-9e5e-16f317da320e)

I have tried troubleshooting suggested in related issues (#65, #110, #37) - 
- Updated docker to latest version,
- Updated Langgraph Studio to latest version, 
- Tried changing shell (currently using `fish`)

```bash
❯ docker version
Client: Docker Engine - Community
 Version:           27.3.1
 API version:       1.47
 Go version:        go1.23.1
 Git commit:        ce1223035a
 Built:             Fri Sep 20 11:01:47 2024
 OS/Arch:           darwin/arm64
 Context:           desktop-linux

Server: Docker Desktop 4.34.3 (170107)
 Engine:
  Version:          27.2.0
  API version:      1.47 (minimum version 1.24)
  Go version:       go1.21.13
  Git commit:       3ab5c7d
  Built:            Tue Aug 27 14:15:41 2024
  OS/Arch:          linux/arm64
  Experimental:     false
 containerd:
  Version:          1.7.20
  GitCommit:        8fc6bcff51318944179630522a095cc9dbf9f353
 runc:
  Version:          1.1.13
  GitCommit:        v1.1.13-0-g58aa920
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0

```

<details>
  <summary>
docker info -f json
  </summary>
  <code>
{
   "Architecture" : "aarch64",
   "BridgeNfIp6tables" : true,
   "BridgeNfIptables" : true,
   "CDISpecDirs" : [],
   "CPUSet" : true,
   "CPUShares" : true,
   "CgroupDriver" : "cgroupfs",
   "CgroupVersion" : "2",
   "ClientInfo" : {
      "Arch" : "arm64",
      "BuildTime" : "Fri Sep 20 11:01:47 2024",
      "Context" : "desktop-linux",
      "Debug" : false,
      "GitCommit" : "ce1223035a",
      "GoVersion" : "go1.23.1",
      "Os" : "darwin",
      "Platform" : {
         "Name" : "Docker Engine - Community"
      },
      "Plugins" : [
         {
            "Name" : "buildx",
            "Path" : "/Users/therexone/.docker/cli-plugins/docker-buildx",
            "SchemaVersion" : "0.1.0",
            "ShortDescription" : "Docker Buildx",
            "Vendor" : "Docker Inc.",
            "Version" : "v0.16.2-desktop.1"
         },
         {
            "Name" : "compose",
            "Path" : "/Users/therexone/.docker/cli-plugins/docker-compose",
            "SchemaVersion" : "0.1.0",
            "ShortDescription" : "Docker Compose",
            "Vendor" : "Docker Inc.",
            "Version" : "v2.29.2-desktop.2"
         },
         {
            "Name" : "debug",
            "Path" : "/Users/therexone/.docker/cli-plugins/docker-debug",
            "SchemaVersion" : "0.1.0",
            "ShortDescription" : "Get a shell into any image or container",
            "Vendor" : "Docker Inc.",
            "Version" : "0.0.34"
         },
         {
            "Name" : "desktop",
            "Path" : "/Users/therexone/.docker/cli-plugins/docker-desktop",
            "SchemaVersion" : "0.1.0",
            "ShortDescription" : "Docker Desktop commands (Alpha)",
            "Vendor" : "Docker Inc.",
            "Version" : "v0.0.15"
         },
         {
            "Name" : "dev",
            "Path" : "/Users/therexone/.docker/cli-plugins/docker-dev",
            "SchemaVersion" : "0.1.0",
            "ShortDescription" : "Docker Dev Environments",
            "Vendor" : "Docker Inc.",
            "Version" : "v0.1.2"
         },
         {
            "Name" : "extension",
            "Path" : "/Users/therexone/.docker/cli-plugins/docker-extension",
            "SchemaVersion" : "0.1.0",
            "ShortDescription" : "Manages Docker extensions",
            "Vendor" : "Docker Inc.",
            "Version" : "v0.2.25"
         },
         {
            "Name" : "feedback",
            "Path" : "/Users/therexone/.docker/cli-plugins/docker-feedback",
            "SchemaVersion" : "0.1.0",
            "ShortDescription" : "Provide feedback, right in your terminal!",
            "Vendor" : "Docker Inc.",
            "Version" : "v1.0.5"
         },
         {
            "Name" : "init",
            "Path" : "/Users/therexone/.docker/cli-plugins/docker-init",
            "SchemaVersion" : "0.1.0",
            "ShortDescription" : "Creates Docker-related starter files for your project",
            "Vendor" : "Docker Inc.",
            "Version" : "v1.3.0"
         },
         {
            "Name" : "sbom",
            "Path" : "/Users/therexone/.docker/cli-plugins/docker-sbom",
            "SchemaVersion" : "0.1.0",
            "ShortDescription" : "View the packaged-based Software Bill Of Materials (SBOM) for an image",
            "URL" : "https://github.com/docker/sbom-cli-plugin",
            "Vendor" : "Anchore Inc.",
            "Version" : "0.6.0"
         },
         {
            "Name" : "scout",
            "Path" : "/Users/therexone/.docker/cli-plugins/docker-scout",
            "SchemaVersion" : "0.1.0",
            "ShortDescription" : "Docker Scout",
            "Vendor" : "Docker Inc.",
            "Version" : "v1.13.0"
         }
      ],
      "Version" : "27.3.1",
      "Warnings" : null
   },
   "ContainerdCommit" : {
      "Expected" : "8fc6bcff51318944179630522a095cc9dbf9f353",
      "ID" : "8fc6bcff51318944179630522a095cc9dbf9f353"
   },
   "Containers" : 2,
   "ContainersPaused" : 0,
   "ContainersRunning" : 0,
   "ContainersStopped" : 2,
   "CpuCfsPeriod" : true,
   "CpuCfsQuota" : true,
   "Debug" : false,
   "DefaultRuntime" : "runc",
   "DockerRootDir" : "/var/lib/docker",
   "Driver" : "overlay2",
   "DriverStatus" : [
      [
         "Backing Filesystem",
         "extfs"
      ],
      [
         "Supports d_type",
         "true"
      ],
      [
         "Using metacopy",
         "false"
      ],
      [
         "Native Overlay Diff",
         "true"
      ],
      [
         "userxattr",
         "false"
      ]
   ],
   "ExperimentalBuild" : false,
   "GenericResources" : null,
   "HttpProxy" : "http.docker.internal:3128",
   "HttpsProxy" : "http.docker.internal:3128",
   "ID" : "2f2c5037-1043-4e3f-911a-adcffd8205d2",
   "IPv4Forwarding" : true,
   "Images" : 7,
   "IndexServerAddress" : "https://index.docker.io/v1/",
   "InitBinary" : "docker-init",
   "InitCommit" : {
      "Expected" : "de40ad0",
      "ID" : "de40ad0"
   },
   "Isolation" : "",
   "KernelVersion" : "6.10.4-linuxkit",
   "Labels" : [
      "com.docker.desktop.address=unix:///Users/therexone/Library/Containers/com.docker.docker/Data/docker-cli.sock"
   ],
   "LiveRestoreEnabled" : false,
   "LoggingDriver" : "json-file",
   "MemTotal" : 8219537408,
   "MemoryLimit" : true,
   "NCPU" : 10,
   "NEventsListener" : 9,
   "NFd" : 47,
   "NGoroutines" : 67,
   "Name" : "docker-desktop",
   "NoProxy" : "hubproxy.docker.internal",
   "OSType" : "linux",
   "OSVersion" : "",
   "OomKillDisable" : false,
   "OperatingSystem" : "Docker Desktop",
   "PidsLimit" : true,
   "Plugins" : {
      "Authorization" : null,
      "Log" : [
         "awslogs",
         "fluentd",
         "gcplogs",
         "gelf",
         "journald",
         "json-file",
         "local",
         "splunk",
         "syslog"
      ],
      "Network" : [
         "bridge",
         "host",
         "ipvlan",
         "macvlan",
         "null",
         "overlay"
      ],
      "Volume" : [
         "local"
      ]
   },
   "RegistryConfig" : {
      "AllowNondistributableArtifactsCIDRs" : null,
      "AllowNondistributableArtifactsHostnames" : null,
      "IndexConfigs" : {
         "docker.io" : {
            "Mirrors" : [],
            "Name" : "docker.io",
            "Official" : true,
            "Secure" : true
         },
         "hubproxy.docker.internal:5555" : {
            "Mirrors" : [],
            "Name" : "hubproxy.docker.internal:5555",
            "Official" : false,
            "Secure" : false
         }
      },
      "InsecureRegistryCIDRs" : [
         "127.0.0.0/8"
      ],
      "Mirrors" : null
   },
   "RuncCommit" : {
      "Expected" : "v1.1.13-0-g58aa920",
      "ID" : "v1.1.13-0-g58aa920"
   },
   "Runtimes" : {
      "io.containerd.runc.v2" : {
         "path" : "runc"
      },
      "runc" : {
         "path" : "runc"
      }
   },
   "SecurityOptions" : [
      "name=seccomp,profile=unconfined",
      "name=cgroupns"
   ],
   "ServerVersion" : "27.2.0",
   "SwapLimit" : true,
   "Swarm" : {
      "ControlAvailable" : false,
      "Error" : "",
      "LocalNodeState" : "inactive",
      "NodeAddr" : "",
      "NodeID" : "",
      "RemoteManagers" : null
   },
   "SystemTime" : "2024-10-20T11:31:21.461632297Z",
   "Warnings" : [
      "WARNING: daemon is not using the default seccomp profile"
   ]
}
</code>
</details>

**Comment by ankit-pn** [comment]
I am getting the same error. Any help will be appreciated.

**Comment by anerathil** [comment]
Same here. Also when starting via zsh (mentioned as solution in a few other tickets) via
`/bin/zsh -c 'open /Applications/LangGraph\ Studio.app/'`

**Comment by anerathil** [comment]
:-O

I just updated Docker and started Langgraph studio via terminal with:

`open /Applications/LangGraph\ Studio.app/`

That worked suprisingly :)

**Comment by shaygenhealth** [comment]
Running into this issue. Updating docker did not fix it. Below is my `docker info -f json`: 

```
{
  "ID": "40a1ced4-6624-4167-8120-aa947723c84e",
  "Containers": 0,
  "ContainersRunning": 0,
  "ContainersPaused": 0,
  "ContainersStopped": 0,
  "Images": 0,
  "Driver": "overlayfs",
  "DriverStatus": [
    [
      "driver-type",
      "io.containerd.snapshotter.v1"
    ]
  ],
  "Plugins": {
    "Volume": [
      "local"
    ],
    "Network": [
      "bridge",
      "host",
      "ipvlan",
      "macvlan",
      "null",
      "overlay"
    ],
    "Authorization": null,
    "Log": [
      "awslogs",
      "fluentd",
      "gcplogs",
      "gelf",
      "journald",
      "json-file",
      "local",
      "splunk",
      "syslog"
    ]
  },
  "MemoryLimit": true,
  "SwapLimit": true,
  "CpuCfsPeriod": true,
  "CpuCfsQuota": true,
  "CPUShares": true,
  "CPUSet": true,
  "PidsLimit": true,
  "IPv4Forwarding": true,
  "BridgeNfIptables": true,
  "BridgeNfIp6tables": true,
  "Debug": false,
  "NFd": 51,
  "OomKillDisable": false,
  "NGoroutines": 77,
  "SystemTime": "2024-10-31T17:53:30.827724594Z",
  "LoggingDriver": "json-file",
  "CgroupDriver": "cgroupfs",
  "CgroupVersion": "2",
  "NEventsListener": 15,
  "KernelVersion": "6.10.4-linuxkit",
  "OperatingSystem": "Docker Desktop",
  "OSVersion": "",
  "OSType": "linux",
  "Architecture": "aarch64",
  "IndexServerAddress": "https://index.docker.io/v1/",
  "RegistryConfig": {
    "AllowNondistributableArtifactsCIDRs": null,
    "AllowNondistributableArtifactsHostnames": null,
    "InsecureRegistryCIDRs": [
      "127.0.0.0/8"
    ],
    "IndexConfigs": {
      "docker.io": {
        "Name": "docker.io",
        "Mirrors": [],
        "Secure": true,
        "Official": true
      },
      "hubproxy.docker.internal:5555": {
        "Name": "hubproxy.docker.internal:5555",
        "Mirrors": [],
        "Secure": false,
        "Official": false
      }
    },
    "Mirrors": null
  },
  "NCPU": 8,
  "MemTotal": 8219820032,
  "GenericResources": null,
  "DockerRootDir": "/var/lib/docker",
  "HttpProxy": "http.docker.internal:3128",
  "HttpsProxy": "http.docker.internal:3128",
  "NoProxy": "hubproxy.docker.internal",
  "Name": "docker-desktop",
  "Labels": [
    "com.docker.desktop.address=unix:///Users/shay/Library/Containers/com.docker.docker/Data/docker-cli.sock"
  ],
  "ExperimentalBuild": false,
  "ServerVersion": "27.2.0",
  "Runtimes": {
    "io.containerd.runc.v2": {
      "path": "runc",
      "status": {
        "org.opencontainers.runtime-spec.features": "{\"ociVersionMin\":\"1.0.0\",\"ociVersionMax\":\"1.0.2-dev\",\"hooks\":[\"prestart\",\"createRuntime\",\"createContainer\",\"startContainer\",\"poststart\",\"poststop\"],\"mountOptions\":[\"acl\",\"async\",\"atime\",\"bind\",\"defaults\",\"dev\",\"diratime\",\"dirsync\",\"exec\",\"iversion\",\"lazytime\",\"loud\",\"mand\",\"noacl\",\"noatime\",\"nodev\",\"nodiratime\",\"noexec\",\"noiversion\",\"nolazytime\",\"nomand\",\"norelatime\",\"nostrictatime\",\"nosuid\",\"nosymfollow\",\"private\",\"ratime\",\"rbind\",\"rdev\",\"rdiratime\",\"relatime\",\"remount\",\"rexec\",\"rnoatime\",\"rnodev\",\"rnodiratime\",\"rnoexec\",\"rnorelatime\",\"rnostrictatime\",\"rnosuid\",\"rnosymfollow\",\"ro\",\"rprivate\",\"rrelatime\",\"rro\",\"rrw\",\"rshared\",\"rslave\",\"rstrictatime\",\"rsuid\",\"rsymfollow\",\"runbindable\",\"rw\",\"shared\",\"silent\",\"slave\",\"strictatime\",\"suid\",\"symfollow\",\"sync\",\"tmpcopyup\",\"unbindable\"],\"linux\":{\"namespaces\":[\"cgroup\",\"ipc\",\"mount\",\"network\",\"pid\",\"user\",\"uts\"],\"capabilities\":[\"CAP_CHOWN\",\"CAP_DAC_OVERRIDE\",\"CAP_DAC_READ_SEARCH\",\"CAP_FOWNER\",\"CAP_FSETID\",\"CAP_KILL\",\"CAP_SETGID\",\"CAP_SETUID\",\"CAP_SETPCAP\",\"CAP_LINUX_IMMUTABLE\",\"CAP_NET_BIND_SERVICE\",\"CAP_NET_BROADCAST\",\"CAP_NET_ADMIN\",\"CAP_NET_RAW\",\"CAP_IPC_LOCK\",\"CAP_IPC_OWNER\",\"CAP_SYS_MODULE\",\"CAP_SYS_RAWIO\",\"CAP_SYS_CHROOT\",\"CAP_SYS_PTRACE\",\"CAP_SYS_PACCT\",\"CAP_SYS_ADMIN\",\"CAP_SYS_BOOT\",\"CAP_SYS_NICE\",\"CAP_SYS_RESOURCE\",\"CAP_SYS_TIME\",\"CAP_SYS_TTY_CONFIG\",\"CAP_MKNOD\",\"CAP_LEASE\",\"CAP_AUDIT_WRITE\",\"CAP_AUDIT_CONTROL\",\"CAP_SETFCAP\",\"CAP_MAC_OVERRIDE\",\"CAP_MAC_ADMIN\",\"CAP_SYSLOG\",\"CAP_WAKE_ALARM\",\"CAP_BLOCK_SUSPEND\",\"CAP_AUDIT_READ\",\"CAP_PERFMON\",\"CAP_BPF\",\"CAP_CHECKPOINT_RESTORE\"],\"cgroup\":{\"v1\":true,\"v2\":true,\"systemd\":true,\"systemdUser\":true},\"seccomp\":{\"enabled\":true,\"actions\":[\"SCMP_ACT_ALLOW\",\"SCMP_ACT_ERRNO\",\"SCMP_ACT_KILL\",\"SCMP_ACT_KILL_PROCESS\",\"SCMP_ACT_KILL_THREAD\",\"SCMP_ACT_LOG\",\"SCMP_ACT_NOTIFY\",\"SCMP_ACT_TRACE\",\"SCMP_ACT_TRAP\"],\"operators\":[\"SCMP_CMP_EQ\",\"SCMP_CMP_GE\",\"SCMP_CMP_GT\",\"SCMP_CMP_LE\",\"SCMP_CMP_LT\",\"SCMP_CMP_MASKED_EQ\",\"SCMP_CMP_NE\"],\"archs\":[\"SCMP_ARCH_AARCH64\",\"SCMP_ARCH_ARM\",\"SCMP_ARCH_MIPS\",\"SCMP_ARCH_MIPS64\",\"SCMP_ARCH_MIPS64N32\",\"SCMP_ARCH_MIPSEL\",\"SCMP_ARCH_MIPSEL64\",\"SCMP_ARCH_MIPSEL64N32\",\"SCMP_ARCH_PPC\",\"SCMP_ARCH_PPC64\",\"SCMP_ARCH_PPC64LE\",\"SCMP_ARCH_RISCV64\",\"SCMP_ARCH_S390\",\"SCMP_ARCH_S390X\",\"SCMP_ARCH_X32\",\"SCMP_ARCH_X86\",\"SCMP_ARCH_X86_64\"]},\"apparmor\":{\"enabled\":true},\"selinux\":{\"enabled\":true}},\"annotations\":{\"io.github.seccomp.libseccomp.version\":\"2.5.4\",\"org.opencontainers.runc.checkpoint.enabled\":\"true\",\"org.opencontainers.runc.commit\":\"v1.1.13-0-g58aa920\",\"org.opencontainers.runc.version\":\"1.1.13\"}}"
      }
    },
    "runc": {
      "path": "runc",
      "status": {
        "org.opencontainers.runtime-spec.features": "{\"ociVersionMin\":\"1.0.0\",\"ociVersionMax\":\"1.0.2-dev\",\"hooks\":[\"prestart\",\"createRuntime\",\"createContainer\",\"startContainer\",\"poststart\",\"poststop\"],\"mountOptions\":[\"acl\",\"async\",\"atime\",\"bind\",\"defaults\",\"dev\",\"diratime\",\"dirsync\",\"exec\",\"iversion\",\"lazytime\",\"loud\",\"mand\",\"noacl\",\"noatime\",\"nodev\",\"nodiratime\",\"noexec\",\"noiversion\",\"nolazytime\",\"nomand\",\"norelatime\",\"nostrictatime\",\"nosuid\",\"nosymfollow\",\"private\",\"ratime\",\"rbind\",\"rdev\",\"rdiratime\",\"relatime\",\"remount\",\"rexec\",\"rnoatime\",\"rnodev\",\"rnodiratime\",\"rnoexec\",\"rnorelatime\",\"rnostrictatime\",\"rnosuid\",\"rnosymfollow\",\"ro\",\"rprivate\",\"rrelatime\",\"rro\",\"rrw\",\"rshared\",\"rslave\",\"rstrictatime\",\"rsuid\",\"rsymfollow\",\"runbindable\",\"rw\",\"shared\",\"silent\",\"slave\",\"strictatime\",\"suid\",\"symfollow\",\"sync\",\"tmpcopyup\",\"unbindable\"],\"linux\":{\"namespaces\":[\"cgroup\",\"ipc\",\"mount\",\"network\",\"pid\",\"user\",\"uts\"],\"capabilities\":[\"CAP_CHOWN\",\"CAP_DAC_OVERRIDE\",\"CAP_DAC_READ_SEARCH\",\"CAP_FOWNER\",\"CAP_FSETID\",\"CAP_KILL\",\"CAP_SETGID\",\"CAP_SETUID\",\"CAP_SETPCAP\",\"CAP_LINUX_IMMUTABLE\",\"CAP_NET_BIND_SERVICE\",\"CAP_NET_BROADCAST\",\"CAP_NET_ADMIN\",\"CAP_NET_RAW\",\"CAP_IPC_LOCK\",\"CAP_IPC_OWNER\",\"CAP_SYS_MODULE\",\"CAP_SYS_RAWIO\",\"CAP_SYS_CHROOT\",\"CAP_SYS_PTRACE\",\"CAP_SYS_PACCT\",\"CAP_SYS_ADMIN\",\"CAP_SYS_BOOT\",\"CAP_SYS_NICE\",\"CAP_SYS_RESOURCE\",\"CAP_SYS_TIME\",\"CAP_SYS_TTY_CONFIG\",\"CAP_MKNOD\",\"CAP_LEASE\",\"CAP_AUDIT_WRITE\",\"CAP_AUDIT_CONTROL\",\"CAP_SETFCAP\",\"CAP_MAC_OVERRIDE\",\"CAP_MAC_ADMIN\",\"CAP_SYSLOG\",\"CAP_WAKE_ALARM\",\"CAP_BLOCK_SUSPEND\",\"CAP_AUDIT_READ\",\"CAP_PERFMON\",\"CAP_BPF\",\"CAP_CHECKPOINT_RESTORE\"],\"cgroup\":{\"v1\":true,\"v2\":true,\"systemd\":true,\"systemdUser\":true},\"seccomp\":{\"enabled\":true,\"actions\":[\"SCMP_ACT_ALLOW\",\"SCMP_ACT_ERRNO\",\"SCMP_ACT_KILL\",\"SCMP_ACT_KILL_PROCESS\",\"SCMP_ACT_KILL_THREAD\",\"SCMP_ACT_LOG\",\"SCMP_ACT_NOTIFY\",\"SCMP_ACT_TRACE\",\"SCMP_ACT_TRAP\"],\"operators\":[\"SCMP_CMP_EQ\",\"SCMP_CMP_GE\",\"SCMP_CMP_GT\",\"SCMP_CMP_LE\",\"SCMP_CMP_LT\",\"SCMP_CMP_MASKED_EQ\",\"SCMP_CMP_NE\"],\"archs\":[\"SCMP_ARCH_AARCH64\",\"SCMP_ARCH_ARM\",\"SCMP_ARCH_MIPS\",\"SCMP_ARCH_MIPS64\",\"SCMP_ARCH_MIPS64N32\",\"SCMP_ARCH_MIPSEL\",\"SCMP_ARCH_MIPSEL64\",\"SCMP_ARCH_MIPSEL64N32\",\"SCMP_ARCH_PPC\",\"SCMP_ARCH_PPC64\",\"SCMP_ARCH_PPC64LE\",\"SCMP_ARCH_RISCV64\",\"SCMP_ARCH_S390\",\"SCMP_ARCH_S390X\",\"SCMP_ARCH_X32\",\"SCMP_ARCH_X86\",\"SCMP_ARCH_X86_64\"]},\"apparmor\":{\"enabled\":true},\"selinux\":{\"enabled\":true}},\"annotations\":{\"io.github.seccomp.libseccomp.version\":\"2.5.4\",\"org.opencontainers.runc.checkpoint.enabled\":\"true\",\"org.opencontainers.runc.commit\":\"v1.1.13-0-g58aa920\",\"org.opencontainers.runc.version\":\"1.1.13\"}}"
      }
    }
  },
  "DefaultRuntime": "runc",
  "Swarm": {
    "NodeID": "",
    "NodeAddr": "",
    "LocalNodeState": "inactive",
    "ControlAvailable": false,
    "Error": "",
    "RemoteManagers": null
  },
  "LiveRestoreEnabled": false,
  "Isolation": "",
  "InitBinary": "docker-init",
  "ContainerdCommit": {
    "ID": "8fc6bcff51318944179630522a095cc9dbf9f353",
    "Expected": "8fc6bcff51318944179630522a095cc9dbf9f353"
  },
  "RuncCommit": {
    "ID": "v1.1.13-0-g58aa920",
    "Expected": "v1.1.13-0-g58aa920"
  },
  "InitCommit": {
    "ID": "de40ad0",
    "Expected": "de40ad0"
  },
  "SecurityOptions": [
    "name=seccomp,profile=unconfined",
    "name=cgroupns"
  ],
  "CDISpecDirs": [],
  "Containerd": {
    "Address": "/run/containerd/containerd.sock",
    "Namespaces": {
      "Containers": "moby",
      "Plugins": "plugins.moby"
    }
  },
  "Warnings": [
    "WARNING: daemon is not using the default seccomp profile"
  ],
  "ClientInfo": {
    "Debug": false,
    "Version": "27.2.0",
    "GitCommit": "3ab4256",
    "GoVersion": "go1.21.13",
    "Os": "darwin",
    "Arch": "arm64",
    "BuildTime": "Tue Aug 27 14:14:45 2024",
    "Context": "desktop-linux",
    "Plugins": [
      {
        "SchemaVersion": "0.1.0",
        "Vendor": "Docker Inc.",
        "Version": "v0.16.2-desktop.1",
        "ShortDescription": "Docker Buildx",
        "Name": "buildx",
        "Path": "/Users/shay/.docker/cli-plugins/docker-buildx"
      },
      {
        "SchemaVersion": "0.1.0",
        "Vendor": "Docker Inc.",
        "Version": "v2.29.2-desktop.2",
        "ShortDescription": "Docker Compose",
        "Name": "compose",
        "Path": "/Users/shay/.docker/cli-plugins/docker-compose"
      },
      {
        "SchemaVersion": "0.1.0",
        "Vendor": "Docker Inc.",
        "Version": "0.0.34",
        "ShortDescription": "Get a shell into any image or container",
        "Name": "debug",
        "Path": "/Users/shay/.docker/cli-plugins/docker-debug"
      },
      {
        "SchemaVersion": "0.1.0",
        "Vendor": "Docker Inc.",
        "Version": "v0.0.15",
        "ShortDescription": "Docker Desktop commands (Alpha)",
        "Name": "desktop",
        "Path": "/Users/shay/.docker/cli-plugins/docker-desktop"
      },
      {
        "SchemaVersion": "0.1.0",
        "Vendor": "Docker Inc.",
        "Version": "v0.1.2",
        "ShortDescription": "Docker Dev Environments",
        "Name": "dev",
        "Path": "/Users/shay/.docker/cli-plugins/docker-dev"
      },
      {
        "SchemaVersion": "0.1.0",
        "Vendor": "Docker Inc.",
        "Version": "v0.2.25",
        "ShortDescription": "Manages Docker extensions",
        "Name": "extension",
        "Path": "/Users/shay/.docker/cli-plugins/docker-extension"
      },
      {
        "SchemaVersion": "0.1.0",
        "Vendor": "Docker Inc.",
        "Version": "v1.0.5",
        "ShortDescription": "Provide feedback, right in your terminal!",
        "Name": "feedback",
        "Path": "/Users/shay/.docker/cli-plugins/docker-feedback"
      },
      {
        "SchemaVersion": "0.1.0",
        "Vendor": "Docker Inc.",
        "Version": "v1.3.0",
        "ShortDescription": "Creates Docker-related starter files for your project",
        "Name": "init",
        "Path": "/Users/shay/.docker/cli-plugins/docker-init"
      },
      {
        "SchemaVersion": "0.1.0",
        "Vendor": "Anchore Inc.",
        "Version": "0.6.0",
        "ShortDescription": "View the packaged-based Software Bill Of Materials (SBOM) for an image",
        "URL": "https://github.com/docker/sbom-cli-plugin",
        "Name": "sbom",
        "Path": "/Users/shay/.docker/cli-plugins/docker-sbom"
      },
      {
        "SchemaVersion": "0.1.0",
        "Vendor": "Docker Inc.",
        "Version": "v1.13.0",
        "ShortDescription": "Docker Scout",
        "Name": "scout",
        "Path": "/Users/shay/.docker/cli-plugins/docker-scout"
      }
    ],
    "Warnings": null
  }
}
```

**Comment by slangenbach** [comment]
I have the same issue.  
Neither updating Docker to the latest version nor opening studio via shell (fish/zsh/bash) did help.

```
~ $ docker version
Client:
 Version:           27.3.1
 API version:       1.47
 Go version:        go1.22.7
 Git commit:        ce12230
 Built:             Fri Sep 20 11:38:18 2024
 OS/Arch:           darwin/arm64
 Context:           desktop-linux

Server: Docker Desktop 4.35.1 (173168)
 Engine:
  Version:          27.3.1
  API version:      1.47 (minimum version 1.24)
  Go version:       go1.22.7
  Git commit:       41ca978
  Built:            Fri Sep 20 11:41:19 2024
  OS/Arch:          linux/arm64
  Experimental:     false
 containerd:
  Version:          1.7.21
  GitCommit:        472731909fa34bd7bc9c087e4c27943f9835f111
 runc:
  Version:          1.1.13
  GitCommit:        v1.1.13-0-g58aa920
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

**Comment by dqbd** [comment]
Hello, released a new 0.0.30 version, which should hopefully address issues with Docker not being detected, especially if the CLI tools are installed in ~/$HOME/.docker/bin and not the system path.

Feel free to reopen if the issue still persists.
